sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -168,4 +168,10 @@ object StaticSQLConf {
       .internal()
       .booleanConf
       .createWithDefault(true)
+
+  val SPARK_SQL_CLI_SHOW_CURRENT_DB_ENABLED =
```

Comment:
This is not worth a new config and code path. user can already show the current DB right?

Reply:
In case of beeline, there is a option to show the current database in the CLI prompt. Same thing I handled for spark-sql CLI

Reply:
beeline is from Hive though. I know we have a script to start it, but I don't think we need to emulate it in this respect.

Reply:
I felt in some cases it’s better if we display the database , for the same reason hive supports it . If you feel this is not required I can close the PR